### ISSUE TITLE
Update European wind capacities

### ISSUE THREAD

**Issue by corradio** [tmrowco/electricitymap#891]
See https://twitter.com/mkaczerowski/status/936997376379772929

**Comment by systemcatch** [comment]
He's saying it's 5800MW, not 6026MW as currently stated?

**Comment by corradio** [comment]
Apparently, but we need to find a solid source.

**Comment by corradio** [comment]
WindEurope's latest report has all capacities. We should probably update all values according to that report.

**Comment by systemcatch** [comment]
[Report](https://windeurope.org/wp-content/uploads/files/about-wind/statistics/WindEurope-Annual-Statistics-2016.pdf) for those who are interested.

**Comment by systemcatch** [comment]
Some of the capacities in the report are lower than the ones we currently have.  Should we update them as well or only do the ones where there is an increase?

**Comment by corradio** [comment]
Let's update all of them I think. @brunolajoie what do you think?

**Comment by brunolajoie** [comment]
It keeps changing anyway, so yeah the best is to wait and update all capacities, once every year or so. I'd rather wait february 2018 for this report to be udpated with 2017 values

**Comment by systemcatch** [comment]
Lets do that then, I've set a reminder for February so it won't be forgotten.  I'll close this for now unless there are any other points to raise?

**Comment by corradio** [comment]
Yep,closing

**Comment by systemcatch** [comment]
The new report is now available https://windeurope.org/wp-content/uploads/files/about-wind/statistics/WindEurope-Annual-Statistics-2017.pdf

**Comment by alixunderplatz** [comment]
@systemcatch I was about to prepare a general update of the latest capacities based on what is on ENTSO-E (when ENTSO-E is used as capacity source). I could also include wind numbers from that report, but it will cause quite a mess in the README to list this as source for wind capacity of all these countries 🤔

**Comment by systemcatch** [comment]
For simplicity I think we'd have one entry in the readme for Wind Europe that lists all countries covered otherwise as you say things will get messy. @brunolajoie do you think it's ok to mix capacity sources like that?

**Comment by brunolajoie** [comment]
The only thing I'm sure is that we can't use ENTSOE capacities without double checking. Initially, I used ENTSOE capacities and then double checked via specialized agencies such as IRENA, or WindEurope, or [OpenPowerSystemData](https://open-power-system-data.org/data-sources#2_Capacity_and_generation_by_technology). When there was too much discrepancy btw ENTSOE and the other (let's say, more than 10%), I trusted the others instead.
